[bug] WikiController 오류 관련 빌드 실패

## ⚠️ 문제 상황
- return SuccessResponse.created(wikiService.createWiki(memberId, request));
  - error: cannot find symbol

## 🔎 예상 결과
<!-- 예상했던 정상적인 결과가 어떤 것이었는지 설명해 주세요 -->

## 💡 원인 및 해결 방안
<!-- 문제 원인과 해결 방안, 참고 자료 등을 첨부해 주세요 -->

--- a/src/main/java/kr/dgucaps/caps/domain/wiki/controller/WikiController.java
+++ b/src/main/java/kr/dgucaps/caps/domain/wiki/controller/WikiController.java
@@ -22,7 +22,7 @@ public class WikiController implements WikiApi {
     @PostMapping
     public ResponseEntity<SuccessResponse<?>> createWiki(@AuthenticationPrincipal(expression = "member") Member member,
                                                          @Valid @RequestBody CreateOrModifyWikiRequest request) {
-        return SuccessResponse.created(wikiService.createWiki(memberId, request));
+        return SuccessResponse.created(wikiService.createWiki(member, request));
     }
 
     @PreAuthorize("hasAnyRole('MEMBER', 'GRADUATE', 'COUNCIL', 'PRESIDENT', 'ADMIN')")

